lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -118,10 +118,8 @@ namespace Js
 #endif
         inlineCacheAllocator(_u("SC-InlineCache"), threadContext->GetPageAllocator(), Throw::OutOfMemory),
         isInstInlineCacheAllocator(_u("SC-IsInstInlineCache"), threadContext->GetPageAllocator(), Throw::OutOfMemory),
-        hasRegisteredInlineCache(false),
-        hasRegisteredIsInstInlineCache(false),
-        entryInScriptContextWithInlineCachesRegistry(nullptr),
-        entryInScriptContextWithIsInstInlineCachesRegistry(nullptr),
+        hasInlineCache(false),
```

Comment:
The condition doesn't mean we have inline cache, but have populated (or in use) inline cache.  
Maybe hasUsedInlineCache?